addons/account/wizard/account_invoice_refund.py

```diff
@@ -130,3 +132,39 @@ def compute_refund(self, mode='refund'):
     def invoice_refund(self):
         data_refund = self.read(['filter_refund'])[0]['filter_refund']
         return self.compute_refund(data_refund)
+
+    @api.multi
+    def compute_charge_note(self, form, inv):
```

Comment:
@umiphos 

Why a new method? and not in the same Odoo method?

Reply:
I tried to mix it with the method of odoo, can you check it? thanks!